samples/word/50-document/manage-custom-xml-part.yaml

```diff
@@ -0,0 +1,222 @@
+order: 7
+id: word-document-manage-custom-xml-part
+name: Manage a CustomXmlPart
+description: 'This sample shows how to add, query, edit, and delete a custom XML part in a document.'
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#add-custom-xml-part").click(() => tryCatch(addCustomXmlPart));
+        $("#query").click(() => tryCatch(query));
+        $("#insert-attribute").click(() => tryCatch(insertAttribute));
+        $("#insert-element").click(() => tryCatch(insertElement));
+        $("#delete-custom-xml-part").click(() => tryCatch(deleteCustomXmlPart));
+
+        async function addCustomXmlPart() {
+          // Add a custom XML part.
+          await Word.run(async (context) => {
+            // You must have the xmlns attribute to populate the CustomXmlPart.namespaceUri property.
```

Comment:
I don't understand this comment. Could you please explain?

Reply:
xmlns was removed from the sample XML in this snippet. Comment removed.